content/metadata.yaml

```
@@ -257,4 +257,14 @@ authors:
   - Department of Bioinformatics and Health Informatics, Children's Hospital of Philadelphia, Philadelphia, PA, 19104, USA
   funders:
   - NCI/NIH Contract No. 75N91019D00024, Task Order No. 75N91020F00003
-
+- github: kelseykeith
+  name: Kelsey Keith
+  initials: KK
+  orcid: 0000-0002-7451-5117
+  email: keithk@chop.edu
+  contributions:
+  - Software
+  - Writing - documentation repository
+  - API
```

These are supposed to follow [credit](https://credit.niso.org/) taxonomy :), but hey, we are making up things as we go here. Will keep it just in case.

```suggestion
  contributions:
  - Software
  - Writing - original draft
  - API
  - Formal Analysis
  - Data Curation
  - Visualization
```